### PR TITLE
Open Source Contributor Initiative Blog Post

### PR DESCRIPTION
**Pull request by rednaksi91**
### Description
Open Source Contributor Initiative Blog Post. Call for applicants.

Note, a link to a form needs to be updated, but other than that the post is ready for review

### Issues Resolved
-

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Comment by rednaksi91** [comment]
@dtaivpp can you please provide feedback if there is any?

File: _posts/2023-06-13-College-Contributor-Initiative.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+layout: post
+title:  "College Contributor Initiative"
+authors:
+  - rednaksi
+date:   2023-06-13 09:00:00 -0700
+categories:
+  - announcements
+meta_keywords: Google Summer of Code, Major League Hacking, MIT Summer Camp, College Contributor Initiative, Coding Bootcamp
+
+meta_description: The OpenSearch Project launched the College Contributor Initiative (CCI) in March 2023, a program that provided students with opportunities to gain software development experience and contribute to open-source projects. The successful 12-week pilot, which concluded in May, led to a significant increase in external contributions, streamlined onboarding for developers, improved code quality, expanded project roadmaps, enhanced open-source maintenance experience, clarified project issues and increased community engagement, prompting an expansion of the program to new schools and locales.
```
Comment (pajuric) [inline _posts/2023-06-13-College-Contributor-Initiative.md:11]: For meta - max is 3-4 targeted keywords/phrases and160 characters with spaces for descriptions. I will provide updated meta description and keywords to ensure they align with our strategy.

**Comment by pajuric** [comment]
@rednaksi91 - Because this was initially a different request ( as webpage) it bypassed being opened as an issue and typically blog process. So, please be sure to open an Asana request for editorial review as I mentioned - until that is done, I cannot move this through the team. I provided the link to you via Slack on 6/12. Reviews can take up to 5-7 working days, so be sure to get the request opened to prevent delays.

File: _posts/2023-06-13-College-Contributor-Initiative.md (hunk 1 above)
Comment (rednaksi91) [reply _posts/2023-06-13-College-Contributor-Initiative.md:11]: @pajuric thank you! When do you think you will be able to provide the updated meta description?

**Review by krisfreedain — CHANGES_REQUESTED** [review]
some changes needed

File: _posts/2023-06-13-Receive-mentorship-from-Amazon-engineers-and-boost-your-portfolio.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
...and scaled the community engagement.** Prior to CCI, activity on the forums was minimal, mostly consisting of communications with known partners. With CCI, the forums saw a burst in activity from contributors helping each other out, and from mentors actively engaging. **Forum posts per month have increased over 40%** since the program, and **replies to posts per month have increased over 85%**. For example, [Alexei Karikov](https://forum.opensearch.org/u/orbulon/summary) joined the OpenSearch forum on Mar 1, and immediately began helping other contributors onboard to the program and overcome their technical challenges. In just two months, he is already **#4 in the** [**Forum’s Global leaderboard**](https://forum.opensearch.org/leaderboard)**,** which includes posts, accepted solutions, and likes. Some repositories that had no prior external engagement saw significant activity. For example, in OpenSearch-Dashboards only 1 pull request came from an external contributor in v2.6.0, jumping to **17 merged pull requests in v2.7.0**. In the opensearch-js client repository, the most recent release had **95% contributions** from student contributors, whereas the previous release had only 2.
```
[the remark this answers is not in the document]
Comment (vagimeli) [reply _posts/2023-06-13-Receive-mentorship-from-Amazon-engineers-and-boost-your-portfolio.md:48]: Do you mean 2%? Or, two contributions. If the latter, we need to be consistent with the data format comparison. For example, "...95% of contributions from student contributors, whereas the previous release had >1%."

**Review by vagimeli — CHANGES_REQUESTED** [review]
Please see edits and comments. Once we address those edits and comments, I'll do a final editorial review. Please reach out if we need to sync on my comments or edits. Thanks, Melissa

**Comment by pajuric** [comment]
@krisfreedain  - This is still working through editorial review cycles.

**Comment by nateynateynate** [comment]
@pajuric , @rednaksi91 - any further submissions here?

**Comment by pajuric** [comment]
@rednaksi91 - Please provide an update in regards to when you will addressing the changes and comments in this draft? Until they are made, we are unable to move this forward. 

cc: @vagimeli @krisfreedain

File: _posts/2023-06-13-Receive-mentorship-from-Amazon-engineers-and-boost-your-portfolio.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+## Amazon’s investment
+
+To support the CCI's pilot phase, 17 AWS engineers from seven participating teams volunteered to be mentors. These mentors establish personal connections and are dedicated to helping contributors succeed in their contributions. They actively create beginner-friendly tasks, promptly review student code, and address student questions. Additionally, the program organizes weekly office hours to offer an alternative avenue for addressing common questions and providing project context. This enables contributors to network and receive expert guidance and personalized feedback from experienced engineers, helping them solve real-world problems. Furthermore, the program offers insights into software development beyond coding, including research, code review, testing, and writing. As we move forward, the program will expand both the number of repository mentors and the number of supported participating repositories, catering to the diverse interests of a wide range of external contributors.
```
Comment (AMoo-Miki) [inline _posts/2023-06-13-Receive-mentorship-from-Amazon-engineers-and-boost-your-portfolio.md:27]:
```suggestion
To support CCI's pilot phase, 17 AWS engineers from seven participating teams volunteered to be mentors. These mentors establish personal connections and are dedicated to helping contributors succeed in their contributions. They actively create beginner-friendly tasks, promptly review student code, and address student questions. Additionally, the program organizes weekly office hours to offer an alternative avenue for addressing common questions and providing project context. This enables contributors to network and receive expert guidance and personalized feedback from experienced engineers, helping them solve real-world problems. Furthermore, the program offers insights into software development beyond coding, including research, code review, testing, and writing. As we move forward, the program will expand both the number of repository mentors and the number of supported participating repositories, catering to the diverse interests of a wide range of external contributors.
```

**Comment by rednaksi91** [comment]
> some changes needed

The Blog post has been re-written and approved by Business, Legal, Marketing and Editorial

File: _posts/2023-09-05-Receive-mentorship-from-Amazon-engineers-and-accelerate-your-career-in-Tech.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+This proverbial "Aerial Balloon Express" is the Open Source Contributor Initiative (OSCI). Spanning 10 weeks, this fully remote program allows participants to receive mentorship from seasoned engineers within a nurturing and inclusive environment that fosters learning and collaboration. Participants in OSCI will work with Amazon engineers and will enhance their portfolios by directly contributing to a prominent open-source software project. When fully leveraged, these benefits can culminate in a recommendation letter signed by an AWS Principal Engineer and an internal endorsement. Imagine dedicating just a few hours each week for 10 weeks and reaping such immense value in return. The best part is that the OSCI program can fit any schedule (even if you’re working or studying full time).
+
+The deadline for applications is **September 13th**, and the program runs **September 20th–November 29th**.
+**Apply now: https://amazonmr.au1.qualtrics.com/jfe/form/SV_esNNNyGLItb5iPY**
```
Comment (krisfreedain) [inline _posts/2023-09-05-Receive-mentorship-from-Amazon-engineers-and-accelerate-your-career-in-Tech.md:23]: you'll want to update so this is hyperlinked: 
`[https://amazonmr.au1.qualtrics.com/jfe/form/SV_esNNNyGLItb5iPY](https://amazonmr.au1.qualtrics.com/jfe/form/SV_esNNNyGLItb5iPY)`

File: _posts/2023-09-05-Receive-mentorship-from-Amazon-engineers-and-accelerate-your-career-in-Tech.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+## **Inviting everyone, everywhere**
+
+OSCI celebrates inclusivity. No boundaries. No borders. We welcome participants globally. Check out our [Code of Conduct](https://opensearch.org/codeofconduct.html) to see exactly how we are making the OpenSearch Project an inclusive and safe environment, or visit our [public Slack channel](https://opensearch.org/slack.html) to meet the community. 
+
+<img src="/assets/media/blog-images/2023-09-05-college-contributor-initiative/Rocket Leaving Earth.jpeg" alt="Kickstart your career"/>{: .img-fluid}
+## **In conclusion**
+
+OSCI will not only help you to grow your technical skills but will also allow you to connect with and learn from experienced Amazon engineers and community members. With guidance and resources at your fingertips, OSCI is poised to shape the builders of tomorrow.
+
+Eager to embark on this journey? **Apply now**: https://amazonmr.au1.qualtrics.com/jfe/form/SV_esNNNyGLItb5iPY
```
Comment (krisfreedain) [inline _posts/2023-09-05-Receive-mentorship-from-Amazon-engineers-and-accelerate-your-career-in-Tech.md:53]: you'll want to update so this is hyperlinked: 
`[https://amazonmr.au1.qualtrics.com/jfe/form/SV_esNNNyGLItb5iPY](https://amazonmr.au1.qualtrics.com/jfe/form/SV_esNNNyGLItb5iPY)`

**Review by krisfreedain — CHANGES_REQUESTED** [review]
a couple tiny edits to make sure your links are actually hyperlinked.

File: _posts/2023-09-05-Receive-mentorship-from-Amazon-engineers-and-accelerate-your-career-in-Tech.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+title:  "Receive mentorship from Amazon engineers and accelerate your career in Tech"
+authors:
+  - rednaksi
+date:   2023-09-05 09:00:00 -0700
+categories:
+  - announcements
+meta_keywords: Open Source Contributor Initiative, Google Summer of Code, Major League Hacking, MIT Summer Camp, Coding Bootcamp
+
+meta_description: The OpenSearch Project launches Open Source Contributor Initiative (OSCI). Spanning 10 weeks, this fully remote program allows participants to receive mentorship from seasoned engineers within a nurturing and inclusive environment. Participants in OSCI will work with Amazon engineers and will enhance their portfolios by directly contributing to a prominent open-source software project. When fully leveraged, these benefits can culminate in a recommendation letter signed by an AWS Principal Engineer and an internal endorsement.
+---
```
Comment (pajuric) [inline _posts/2023-09-05-Receive-mentorship-from-Amazon-engineers-and-accelerate-your-career-in-Tech.md:12]: Please update the meta with the following: 
**meta_keywords:** Open Source College Contributor Initiative, open source software project, OpenSearch college initiative
**meta_description:** Join the OpenSearch Project’s College Contributor Initiative and receive mentorship while collaborating closely with AWS engineers and growing your open-source project knowledge.

**Comment by pajuric** [comment]
@krisfreedain @nateynateynate @dtaivpp - We're are good to publish this today.

**Review by nateynateynate — APPROVED** [review]
Looks fine to me.

**Review by nateynateynate — APPROVED** [review]
I took the liberty of updating the meta information. Let's get it out!

**Review by nateynateynate — APPROVED** [review]
Ship it.

**Pull request merged**